tgui/packages/tgui-panel/stat/StatText.js

```diff
@@ -61,19 +63,62 @@ export const StatTextText = (props, context) => {
 };
 
 export const StatTextButton = (props, context) => {
-  const { title, text, action_id, params = [] } = props;
+  const { title, text, action_id, params = [], multirow = false, buttons = [] } = props;
   return (
     <Flex.Item mt={1}>
       <Button
+        width="100%"
+        overflowX="hidden"
         onClick={() =>
           Byond.sendMessage('stat/pressed', {
             action_id: action_id,
             params: params,
           })
         }
         color="transparent">
-        <b>{title}: </b>
-        {text}
+        {!multirow ? (
+          <>
+            <Box bold inline mr="5px" verticalAlign="top">
+              {title}
+            </Box>
+            <Box
+              width="100%"
+              pr="80px"
+              inline
+              style={{
+                'white-space': 'normal',
+              }}>
+              {text}
+            </Box>
+          </>
+        ) : (
+          <>
+            <Flex bold direction="row">
+              <Flex.Item grow={1}>{title}</Flex.Item>
+              {buttons.map((buttonInfo) => (
```

Comment:
extra buttons should be supported on non multirow elements

Reply:
I was going to add this but its not easy to make this look good. Could probably just shove it in flex and let that deal with sizing.